### ISSUE TITLE
create tombstone page for DOIs we need to remove

### ISSUE THREAD

**Issue by jeanetteclark** [NCEAS/metacat#1413]
From the latest EZID report, there are a bunch of bad links to DOIs from removing some of the WT tests that made it to production: https://support.nceas.ucsb.edu/rt/Ticket/Display.html?id=19456

We should build a tombstone page for these DOIs

**Comment by mbjones** [comment]
Agreed.  We also have a few broken due to us restucturing some of our plone websites, so their URIs changed.  I have a request in with EZID to update those because they don't seem to be editable from our account.  So let's be careful that the tombstone page isn't used for DOIs that should point at new locations.
